src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseAgreedRepaymentRespondentNotificationHandler.java

```diff
@@ -81,7 +81,7 @@ CLAIM_DEFENDANT_LEGAL_ORG_NAME_SPEC, getRespondentLegalOrganizationName(
                 CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference()
             );
         }
-        if (isRespondentNotRepresented(caseData)) {
+        if (caseData.isRespondentResponseBilingual()) {
```

Comment:
are you sure you are not introducing a bug here? don't we need those for when the response is not biligual?

Reply:
Well spotted, fixed